src/axom/quest/examples/quest_signed_distance_interface.cpp

```diff
@@ -134,10 +139,10 @@ struct Arguments
     std::string pol_info =
       "Sets execution space of the SignedDistance query.\n";
     pol_info += "Set to \'seq\' to use sequential execution policy.";
-#ifdef AXOM_USE_OPENMP
+#if defined(AXOM_USE_OPENMP) && defined(AXOM_USE_RAJA)
```

Comment:
Note: `OpenMP` capability of `SignedDistance` query requires `RAJA` but was missing preprocessor guards